README.md

```diff
@@ -89,30 +97,33 @@ $multipleValidations = new MultipleValidationWithAnd([
 $validator->isValid("example@ietf.org", $multipleValidations); //true
 ```
 
-#### Additional validations ####
-Validations not present in the RFCs 
+#### Additional validations
 
-1. [SpoofCheckValidation](/src/Validation/Extra/SpoofCheckValidation.php): Will check for multi-utf-8 chars that can signal an erroneous email name.
+Validations not present in the RFCs
 
+1. [SpoofCheckValidation](/src/Validation/Extra/SpoofCheckValidation.php): Will check for multi-utf-8 chars that can
+   signal an erroneous email name.
 
-### How to extend ###
+### How to extend
 
-It's easy! You just need to implement [EmailValidation](/src/Validation/EmailValidation.php) and you can use your own validation.
+It's easy! You just need to implement [EmailValidation](/src/Validation/EmailValidation.php) and you can use your own
```

Comment:
Undo this change please.

Reply:
Refer to https://github.com/egulias/EmailValidator/pull/364#discussion_r1212802161.